scripts/nagios/check-rabbitmq-queue

```diff
@@ -31,50 +36,147 @@ states = {
     3: "UNKNOWN"
 }
 
-pattern = re.compile(r'(\w+)\t(\d+)')
-output = subprocess.check_output(['/usr/sbin/rabbitmqctl', 'list_queues'], universal_newlines=True)
+MAX_SECONDS_TO_CLEAR_FOR_BURSTS = 600
+MAX_SECONDS_TO_CLEAR_NORMAL = 120
+CRITICAL_SECONDS_TO_CLEAR_FOR_BURSTS = 1200
```

Comment:
I think these values should have a model more like the `CRIT_THRESHOLD` where we can override them for particular queues.  These numbers are appropriate for `digest_emails`, but like 4x too high for most other queues.

Reply:
Done! It's more configurable now, but I'm guessing that values I've set won't be what we ultimately want.